src/m365/tenant/commands/people/people-profilecardproperty-get.ts

```diff
@@ -0,0 +1,92 @@
+import { Logger } from '../../../../cli/Logger.js';
+import GlobalOptions from '../../../../GlobalOptions.js';
+import GraphCommand from '../../../base/GraphCommand.js';
+import request, { CliRequestOptions } from '../../../../request.js';
+import { profileCardPropertyNames, ProfileCardProperty } from './profileCardProperties.js';
+import commands from '../../commands.js';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  name: string;
+}
+
+class TenantPeopleProfileCardPropertyGetCommand extends GraphCommand {
+  public get name(): string {
+    return commands.PEOPLE_PROFILECARDPROPERTY_GET;
+  }
+
+  public get description(): string {
+    return 'Retrieves information about a specific profile card property';
+  }
+
+  constructor() {
+    super();
+
+    this.#initOptions();
+    this.#initValidators();
+  }
+
+  #initOptions(): void {
+    this.options.unshift(
+      {
+        option: '-n, --name <name>',
+        autocomplete: profileCardPropertyNames
+      }
+    );
+  }
+
+  #initValidators(): void {
+    this.validators.push(
+      async (args: CommandArgs) => {
+        if (!profileCardPropertyNames.some(p => p.toLowerCase() === args.options.name.toLowerCase())) {
+          return `'${args.options.name}' is not a valid value for option name. Allowed values are: ${profileCardPropertyNames.join(', ')}.`;
+        }
+
+        return true;
+      }
+    );
+  }
+
+  public async commandAction(logger: Logger, args: CommandArgs): Promise<void> {
+    try {
+      if (this.verbose) {
+        await logger.logToStderr(`Retrieving information about profile card property '${args.options.name}'...`);
+      }
+
+      const requestOptions: CliRequestOptions = {
+        url: `${this.resource}/v1.0/admin/people/profileCardProperties/${args.options.name}`,
+        headers: {
+          accept: 'application/json;odata.metadata=none'
+        },
+        responseType: 'json'
+      };
+
+      const result = await request.get<ProfileCardProperty>(requestOptions);
+      let output: any = result;
+
+      // Transform the output to make it more readable
+      if (args.options.output && args.options.output !== 'json' && result.annotations.length > 0) {
+        output = result.annotations[0].localizations.reduce((acc, curr) => ({
```

Comment:
A slight downside here: any new root properties that microsoft adds will not be visible in the eventual output.

What I did myself was just edit the response object slightly.  

```typescript
      // Transform the output to make it more readable
      if (args.options.output && args.options.output !== 'json' && response.annotations.length > 0) {
        const annotation = response.annotations[0];

        response.displayName = annotation.displayName;
        annotation.localizations.forEach((l: { languageTag: string, displayName: string }) => {
          response[`displayName ${l.languageTag}`] = l.displayName;
        });

        delete response.annotations;
        delete response['@odata.context'];
      }
```